src/model/rest/nn_seeker_paservice_show.py

```diff
@@ -0,0 +1,42 @@
+import logging
```

Comment:
For my understanding - why do we need different classes for `nn_seeker_paservice` and `nn_seeker_paservice_show`? This looks a little convolutet - also given that `nn_seeker_paservice_show` inherits from `nn_seeker_paservice`.  I leave it up to you if you want to adjust it or not.

Reply:
I think for now this is the cleanest way of doing it, since we need this extra lookup from show to first eposide. Once we have proper show metadata, things might look different, but we don't